lib/waterline/query/finders/basic.js

```diff
@@ -45,6 +45,9 @@ module.exports = {
     // Normalize criteria
     criteria = normalize.criteria(criteria);
 
+    // Use criteria defaults, if any
+    utils.mergeCriteria(criteria,self.defaultCriteria);
```

Comment:
Any reason we don't do `criteria = utils.mergeCriteria(criteria,self.defaultCriteria);` like they do `criteria = normalize.criteria(criteria);`?

Would be a little easier to understand I think.